### ISSUE TITLE
[FEATURE] Playwright-axe

### ISSUE THREAD

**Issue by Scc33** [Scc33/Scc33.github.io#118]
**Describe the solution you'd like**
Playwright should be used to check for Accessibility violations

**Additional context**
- https://www.npmjs.com/package/axe-playwright
- https://playwright.dev/docs/accessibility-testing
- Split from #100
